packages/help-center/src/hooks/use-context-based-search-mapping.tsx

```diff
@@ -22,15 +24,21 @@ export function useContextBasedSearchMapping( currentRoute: string | undefined )
 		return '';
 	}, [] );
 
+	// Fuzzier matches
+	const urlMatchKey = Object.keys( urlMapping ).find( ( key ) => currentRoute?.startsWith( key ) );
+	const urlSearchQuery = urlMatchKey ? urlMapping[ urlMatchKey as keyof typeof urlMapping ] : '';
+	const tailoredArticles = urlMatchKey
+		? tailoredArticlesMapping[ urlMatchKey as keyof typeof tailoredArticlesMapping ]
+		: [];
+
```

Comment:
I don't think this works if we have an address in the tailored articles and not in urlMapping! `urlMatchKey` would not have any value.

Reply:
@escapemanuele - Do you mean if I remove, `"/home/": "home"` (example), from the route-to-query-mapping.json, we won't see tailored articles?

If im understanding correctly, then yes I don't see any tailored results that mapping.